Client/src/components/common/Table/index.css

```diff
@@ -0,0 +1,33 @@
+.Table {
+  margin: 50px auto;
+  border-spacing: 0px;
+  background: #fff;
+  box-shadow: 0 1px 0 0 rgba(22, 29, 37, 0.05);
+  font-size: 14px;
+  width: 80%;
```

Comment:
that should be ```  100%  ```
because when I will use your table I will wrap it in div and align it where i want